Cargo.toml

```diff
@@ -19,9 +19,6 @@ substreams = "0.3.2"
 substreams-ethereum = "0.6.2"
 substreams-solana = "0.1.0"
 
-[build]
-target = "wasm32-unknown-unknown"
```

Comment:
why remove this?

Reply:
for some reason I couldn't build with it in there. But I fixed it